db/migrate/20210928071137_update_subject_codes.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateSubjectCodes < ActiveRecord::Migration[6.1]
+  def change
+    subject_category = SubjectCategory.first
+    return if subject_category.nil?
+
+    Subject.create(code: "ARNT", title: "Arithmetic Number Theory", subject_category_id: subject_category.id)
```

Comment:
This will create duplicates if it already exists. Please search first and return if exists.